pkg/controller/quayecosystem/provisioning/provision.go

```diff
@@ -927,7 +927,7 @@ func (r *ReconcileQuayEcosystemConfiguration) ManageQuayEcosystemCertificates(me
 
 			clairSslSecret = resources.GetTLSSecretDefinition(meta, privKeyBytes, certBytes)
 
-			err = r.reconcilerBase.CreateOrUpdateResource(r.quayConfiguration.QuayEcosystem, r.quayConfiguration.QuayEcosystem.Namespace, clairSslSecret)
+			err = r.reconcilerBase.CreateResourceIfNotExists(r.quayConfiguration.QuayEcosystem, r.quayConfiguration.QuayEcosystem.Namespace, clairSslSecret)
```

Comment:
Found a bug where if the operator triggered twice it would create two instances of the same pod. I believe the desired functionality is to have 1 pod, @sabre1041 please review.

Reply:
Would you be able to elaborate?